package Lectures.JAVA;
import java.util.Scanner;
public class exceptionHandling {
    // public static void main(String[] args) {
    //  Scanner sc = new Scanner(System.in);
  /*       int a = sc.nextInt();
           int b = sc.nextInt();
     try{
         int c = a / b;
         System.out.println(c);
     }
     catch(Exception e){
         System.out.println("The problem is: " + e);
     }
     */
   /*
    int [] arr = new int[4];
     arr[0]=9;
     arr[1]=5;
     arr[2]=6;
     arr[3]=7;
     try{
         System.out.print("Index: ");
         int q = sc.nextInt();
         System.out.print("Number to divide: ");
         int p = sc.nextInt();
         int z = arr[q]/p;
     }
     catch(IndexOutOfBoundsException e){
         System.out.println("Error of index!!");
     }
     catch(ArithmeticException e){
         System.out.println("Error of ArithmeticExp");
     }
     */
       /* int [] arr = new int[4];
        arr[0]=9;
        arr[1]=5;
        arr[2]=6;
        arr[3]=7;
    while(true) {
        try {
            System.out.println("Niche dekh lawde!!");
            try {
                int l = sc.nextInt();
                System.out.println(arr[l]);
            } catch (IndexOutOfBoundsException e) {
                System.out.println("Error at lvl 2");
            }
        } catch (Exception e) {
            System.out.println("Error at lvl 1");
        }
    }*/

    public static void main(String[] args) {

        }
    }
